### DISCUSSION TITLE
How to use filter markers?

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#60] [Q&A]
Thanks for the update，3.10.0 has a surprising update.
How to use filter markers?

**Comment by zhubib1234** [comment]
![Uploading 2.png…]()

**Reply by valentine195** [reply]
Sorry- your image didn't upload properly for me. 

You should see a filter button in the top right off the map. Clicking on it shows the active marker types on the map. You can toggle them off to disable all markers of that type.

**Comment by zhubib1234** [comment]
![2](https://user-images.githubusercontent.com/84388916/119057359-e7663900-b9fe-11eb-9789-34eeb8fbfd74.png)

**Reply by valentine195** [reply]
Hmm, does that map have any markers defined?

If so, could you open the console (control-shift-i / ctrl- cmd-i) and tell me if there are any errors.

**Reply by zhubib1234** [reply]
I set up two kinds of marker.

**Comment by zhubib1234** [comment]
![Uploading 3.png…]()

**Comment by zhubib1234** [comment]
![3](https://user-images.githubusercontent.com/84388916/119059358-b556d600-ba02-11eb-9a75-38c331eec1c5.png)

**Reply by valentine195** [reply]
That specific error looks like it is related to trying to zoom to show all markers when there are no markers created on the map. Thanks for letting me know - will be fixed in the next release.

As far as the empty filter box: the filter control box displays the marker types that are _actively on the map_.

For example:

You have created three marker types:
1. Location
2. Person
3. Event

On a map, you have 5 markers: 3 location type markers, 2 event type markers. The control box will display Location and Events as types you can filter on or off.

On a different map, you only have Person markers. The control box will only display Person as a type you can turn on or off.
